files/nginx/grafana.conf

```diff
@@ -0,0 +1,15 @@
+server {
+    listen      80;
+    listen      [::]:80 ipv6only=on;
```

Comment:
What does ipv6only=on do ?

Reply:
it tell nginx to only use this line if it's receiving an ipv6 address

Reply:
I don't think this is needed anymore in recent versions of NGinx.
https://serverfault.com/questions/842492/nginx-duplicate-listen-options-for-80-error/842515